src/inspector/view/qml/MuseScore/Inspector/text/TextInspectorView.qml

```diff
@@ -93,24 +93,26 @@ InspectorSectionView {
 
                     model: [
                         { iconCode: IconCode.TEXT_BOLD, value: TextTypes.FONT_STYLE_BOLD, title: qsTrc("inspector", "Bold") },
-                        { iconCode: IconCode.TEXT_ITALIC, value: TextTypes.FONT_STYLE_ITALIC, title: qsTrc("inspector", "Italic") },
+                        { iconCode: IconCode.TEXT_ITALIC, value: TextTypes.FONT_STYLE_ITALIC, title: qsTrc("inspector", "Italics") },
```

Comment:
Sorry, my mistake. "Italic" (without the 's') was correct. I got it wrong in https://github.com/musescore/MuseScore/issues/13012#issuecomment-1227272646.